bigtable/admin.go

```diff
@@ -1663,3 +1664,53 @@ func (ac *AdminClient) UpdateBackup(ctx context.Context, cluster, backup string,
 	_, err = ac.tClient.UpdateBackup(ctx, req)
 	return err
 }
+
+// GetIamPolicy gets the IAM access control policy for the specified backup.
+func (ac *AdminClient) GetIamPolicy(ctx context.Context, cluster, backup string) (*iampb.Policy, error) {
```

Comment:
Looks like this should be named `BackupIAM` to match with `TableIAM`: https://pkg.go.dev/cloud.google.com/go/bigtable#AdminClient.TableIAM

Reply:
Thanks for quick response! 
Hmm, I thought we should have three methods, like in other languages, but your suggestion makes more sense.
Not a problem, I'll update.